types/defines/queues.d.ts

```diff
@@ -14,6 +14,14 @@ interface Message<Body = unknown> {
    * The body of the message.
    */
   readonly body: Body;
+  /**
+   * Marks message to be retried.
+   */
+  retry(): void;
+  /**
+   * Marks message acked.
```

Comment:
I'd be extra explicit and use "acknowledged" rather than "acked" in case someone is looking at this comment specifically because they don't know what "ack" means (both here and on ackAll below).

Otherwise this LGTM.